cron/monthly/invoice-platform-fees.js

```diff
@@ -338,13 +342,8 @@ export async function run() {
     }
 
     const transactionIds = transactions.map(t => t.TransactionId);
-    const totalAmountCredited = sumBy(
-      items
-        .filter(i => i.description != 'Shared Revenue')
-        .filter(i => i.description != 'Reimburse: Payment Processor Fee for collected Platform Tips')
-        .filter(i => i.description != 'Fixed Fee per Hosted Collective'),
-      'amount',
-    );
+    const itemsWithoutDebt = items.filter(i => i.description === 'Platform Fees');
+    const totalAmountCredited = sumBy(itemsWithoutDebt, 'amount');
```

Comment:
It's a general comment about this file. I would like us to stop doing custom queries in SQL and use instead some helpers in the model or in separated library.

The goal would be to have a single source of truth, and share metrics with:
https://github.com/opencollective/opencollective-api/blob/main/server/models/Collective.js#L2946